sys/include/net/gnrc/netif/conf.h

```diff
@@ -189,6 +189,18 @@ extern "C" {
 #define CONFIG_GNRC_NETIF_NONSTANDARD_6LO_MTU 0
 #endif
 
+/**
+ * @brief   Enable the usage of non standard MTU for IPv6 network interfaces
+*
+ * @experimental
+ *
+ * This feature is non compliant with RFC 7668 and might not be supported by
+ * other implementations.
+ */
+#ifndef CONFIG_GNRC_NETIF_NONSTANDARD_IPV6_MTU
+#define CONFIG_GNRC_NETIF_NONSTANDARD_IPV6_MTU 0
```

Comment:
Why not re-use `CONFIG_GNRC_NETIF_NONSTANDARD_6LO_MTU` above? IPv6 over BLE is still under the 6Lo-umbrella ;-)